concrete-core/docs/dev/index.rst

```diff
@@ -0,0 +1,16 @@
+Contribution guides
+===================
+
+.. (Allocateurs dans l´engine, conversion pour transfert de données, plusieurs engines pour plusieurs algo, destructeurs.)
+.. (The tutorial showing how to write a backend. It would be nice to split the tutorial into multiple subpages)
+.. (A tutorial explaining how to benchmark a backend with `concrete-bench`)
+.. (Benchmarking a backend ./benchmarking_backends.md)
+.. (A tutorial explaining how to test a backend with `concrete-test`)
+
+.. toctree::
+   :maxdepth: 2
+
+   architecture.md
```

Comment:
should we keep empty parts?

Reply:
This was due to the former mdbook way to handle the TOC, I believe. I am not sure that makes sense for sphinx !